lms/envs/common.py

```diff
@@ -1917,6 +1922,15 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+#################### Badgr OpenBadges generation #######################
+BADGR_API_TOKEN = None
+BADGR_BASE_URL = "http://localhost:8005"
+BADGR_ISSUER_SLUG = "example-issuer"
+# A dictionary of course modes to file paths. Example:
+# {'honor': '/path/to/badge_image.png'}
+# Badge images must be square and in PNG or SVG format.
+BADGR_IMAGE_SOURCES = {}
```

Comment:
The current API doesn't let you update badges, and you can only delete them if they haven't been issued. If you make a mistake defining a badge, delete it before it has been issued and re-create it. 

As far as image storage for defined badges and issued (baked) badge images, the next upgrade to the Issuer server will allow you to use an AWS S3 bucket to store images, so they can be accessible across several application servers.

PNG support is all there is for now, but SVG will be coming this summer.